test/test_torch.py

```diff
@@ -4240,6 +4240,40 @@ def test_svd(self):
         Xhat = torch.mm(U, torch.mm(S.diag(), V.t()))
         self.assertEqual(X, Xhat, 1e-8, 'USV\' wrong')
 
+    @staticmethod
+    def _test_matrix_rank(self, conv_fn):
```

Comment:
Sorry, it seems that conv_fn isn't used. Can you fix it please? Or better, switch to a device object?

Reply:
I have fixed it, thank you for letting me know. I keep forgetting about `conv_fn`. Sorry.